mcs/class/System.Windows.Forms/System.Windows.Forms/Control.cs

```diff
@@ -5395,6 +5395,12 @@ private void WmLButtonDown (ref Message m)
 		}
 
 		private void WmMButtonDown (ref Message m) {
+			// Menu handle.
+			if (XplatUI.IsEnabled (Handle) && active_tracker != null) {
+				ProcessActiveTracker (ref m);
+				return;
+			}
```

Comment:
should it be added to WmMButtonUp too? both WmLButtonUp and WmRButtonUp have it.

Reply:
You're right, it should be added to WmMButtonUp too.